js/src/logger.ts

```diff
@@ -719,22 +719,40 @@ function constructLogs3Data(items: string[]) {
   return `{"rows": ${constructJsonArray(items)}, "api_version": 2}`;
 }
 
-const DefaultBatchSize = 100;
-const NumRetries = 3;
-
 function now() {
   return new Date().getTime();
 }
 
 class BackgroundLogger {
   private logConn: LazyValue<HTTPConnection>;
   private items: LazyValue<BackgroundLogEvent>[] = [];
-  private active_flush: Promise<string[]> = Promise.resolve([]);
-  private active_flush_resolved = true;
+  private activeFlush: Promise<void> = Promise.resolve();
+  private activeFlushResolved = true;
+
+  public syncFlush: boolean = false;
+  public defaultBatchSize: number = 100;
+  public numRetries: number = 3;
 
   constructor(logConn: LazyValue<HTTPConnection>) {
     this.logConn = logConn;
 
+    const syncFlushEnv = Number(iso.getEnv("BRAINTRUST_SYNC_FLUSH"));
```

Comment:
should we document these variables somewhere? Maybe in a troubleshooting section of the eval guide?

Reply:
Yes. Will add in the alignment PR.